src/onboarding-component/index.jsx

```diff
@@ -103,9 +108,23 @@ export const OnBoardingComponent = ({
     if (context) {
       validatedContext = validateContextData(context);
     }
+    if (isAuthenticatedUser) {
+      const queryParamString = objectToQueryString({ ...queryParams, ...onboardingComponentContext });
+      if (formToRender === REGISTRATION_FORM) {
+        window.location.href = `${getConfig().LMS_BASE_URL}/register?${queryParamString}`;
+      } else {
+        window.location.href = `${getConfig().LMS_BASE_URL}/${formToRender}?${queryParamString}`;
+      }
```

Comment:
We can simply the condition

```suggestion
  const formUrl = formToRender === REGISTRATION_FORM ? 'register' : formToRender;
  window.location.href = `${getConfig().LMS_BASE_URL}/${formUrl}?${queryParamString}`;
```

Reply:
done